### ISSUE TITLE
Oscillatory sensation

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6806]
Preferred term label:
Oscillatory sensation

Definition (free text, please give PubMed ID)
PMID: 32986636

Parent term (use hpo.jax.org/app)
No found because of the specificity of the disease

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
ORPHA:210272

**Comment by pnrobinson** [comment]
adding as
HP:4000033
Non-spinning vertigo
def:A perception of unsteadiness of dizziness that is not characterized bythe spinning sensation of classic vertigo, but rather by an oscillatory perception (rocking, bobbing, or swaying despite the fact that the subject is not moving).
comment: This feature can be observed in mal de debarquement syndrome and in vestibular paroxysmia.
